src/mlpack/methods/ann/layer/sigmoid_cross_entropy_error.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file sigmoid_cross_entropy_error.hpp
+ * @author Kris Singh
+ *
+ * Definition of the cross-entropy with logit performance function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_SIGMOID_CROSS_ENTROPY_ERROR_HPP
+#define MLPACK_METHODS_ANN_LAYER_SIGMOID_CROSS_ENTROPY_ERROR_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The SigmoidCrossEntropyError performance function measures the network's
+ * performance according to the cross-entropy function.
+ * between the input and target distributions.
+ * This function calculates the cross entropy
+ * given the real values instead of providing the sigmoid activations.
+ * The functions is much more numerically stable as can be found from the 
+ * formula below.
+ * max(x, 0) - x * z + log(1 + exp(-abs(x)))
```

Comment:
Can you use the doxygen forumla syntax here, so that we can render the expression? For more information take a look at: https://www.stack.nl/~dimitri/doxygen/manual/formulas.html